exps/default/yolov3.py

```diff
@@ -33,10 +33,11 @@ def init_yolo(M):
         return self.model
 
     def get_data_loader(self, batch_size, is_distributed, no_aug=False):
-        from data.datasets.cocodataset import COCODataset
-        from data.datasets.mosaicdetection import MosaicDetection
-        from data.datasets.data_augment import TrainTransform
-        from data.datasets.dataloading import YoloBatchSampler, DataLoader, InfiniteSampler
+        from yolox.data.datasets.coco import COCODataset
```

Comment:
Thx~Could you please merge the import to a few lines, e.g. from yolox.data import blabla.

Reply:
okok